### PR TITLE
fix(@angular/cli): adding multiple apps story

### PR DESCRIPTION
**Pull request by sumitarora**
Fixes: https://github.com/angular/angular-cli/issues/5726

### CODE REVIEW

File: docs/documentation/stories/multiple-apps.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+      "prod": "environments/environment.prod.ts"
+    }
+  }  
+],
+```
+Now we can `serve`, `build` etc. both the apps by passing the app index with the commands. By default, it will pick the first app only.
+
+To serve the first app: `ng serve --app=0` or `ng serve --app 0`
+
+To serve the second app: `ng serve --app=1` or `ng serve --app 12`
```
Comment (intellix) [inline docs/documentation/stories/multiple-apps.md]: 12? :D

Comment (sumitarora) [inline docs/documentation/stories/multiple-apps.md]: @intellix Fixed

**Comment by sumitarora** [comment]
@Brocco done the changes.

**Comment by realappie** [comment]
@Brocco Is there any way to ensure the angular cli generators still work with multiple apps? Im pretty sure you're familiar with the following error
```
Error locating module for declaration
SilentError: No module files found
```

According to a few github issues I read it has to do with multiple app module files being in the src directory, is there any recommended way to have a multiple apps setup without breaking the CLI generators?

**Comment by cvgaviao** [comment]
Hi, I'm reading [#7418](https://github.com/angular/angular-cli/issues/7418) and there is a comment there that say that in order to use multiple app in same project we need to create a dir src/new-app and copy every file under the src/app to it. 
But I can't find any info in the [stories-multiple-apps](https://github.com/angular/angular-cli/wiki/stories-multiple-apps) about what should/must be copied and how to set "root" and "outDir" properly. 
Could someone put the right approach info there ?

**Pull request merged**